ngraph/python/tests/test_ngraph/test_create_op.py

```diff
@@ -1096,6 +1170,28 @@ def test_tensor_iterator():
     assert list(node.get_output_shape(1)) == [16, 2, 2]
 
 
+def test_read_value_opset5():
+    init_value = ng_opset5.parameter([2, 2], name="init_value", dtype=np.int32)
+
+    node = ng_opset5.read_value(init_value, "var_id_667")
```

Comment:
Isn't this op added to `opset6` init? Why we use `opset5` here?

Reply:
The answer is below